.travis.yml

```diff
@@ -9,6 +9,8 @@ services:
 # have more than one run with the same node version
 # otherwise they will clobber each other
 #####
+env:
+  node_pre_gyp_mock_s3=true
```

Comment:
This is great. Even better I think would be another matrix/job so that:

 - We test both mocking and no mocking
 - Only the mocking job is used with coverage is tested and uploaded

Sound good? If so, could you add another job that uses `node_pre_gyp_mock_s3=false`? You should be able to do this by moving the `env` into each matrix item.

Reply:
it's easy enough to move the env into the matrix and add the `false` env setting.

but i'm not sure what you mean by "only the mocking job is with coverage is tested and uploaded".

and maybe it's related, but if mocking is false won't all external PR tests always fail?